planemo/shed/__init__.py

```diff
@@ -144,6 +144,7 @@
     "RNA",
     "SAM",
     "Sequence Analysis",
+    "Single Cell",
```

Comment:
```suggestion
    "Single Cell",
    "Spatial Omics",
```

And can you also add `"Muon spectroscopy",` please?